### PR TITLE
Monitor updates once every minute

### PR DESCRIPTION
**Pull request by Elfoniok**

### CODE REVIEW

File: golem/ethereum/paymentprocessor.py [hunk 1]
```diff
@@ -379,7 +379,6 @@ def _run(self):
 
             self.monitor_progress()
             self.sendout()
-            self._send_balance_snapshot()
```
Comment (Wiezzel) [inline golem/ethereum/paymentprocessor.py:382]: Why? Isn't it already disabled by `send_payment_info` flag in monitor?

Comment (Elfoniok) [reply golem/ethereum/paymentprocessor.py:382]: Oh it is but only for mainnet, i didn't noticed this. But imho if we are not want this updates we should not even bother dispatcher with sending them.

Comment (Wiezzel) [reply golem/ethereum/paymentprocessor.py:382]: It's not that we don't want them. We don't want them now, but we want this to reconfigurable. Removing this line basically makes the flag pointless.

Comment (Elfoniok) [reply golem/ethereum/paymentprocessor.py:382]: Because this flag is in wrong place after all

Comment (Wiezzel) [reply golem/ethereum/paymentprocessor.py:382]: OK, so maybe just move the flag to payment processor?

Comment (Elfoniok) [reply golem/ethereum/paymentprocessor.py:382]: Maybe, its not the point of this change. This is a quick fix for monitor.

**Review by Wiezzel — APPROVED** [review]

**Review by jiivan — APPROVED** [review]

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**